.mvn/gradle-enterprise.xml

```diff
@@ -0,0 +1,55 @@
+<gradleEnterprise>
+  <server>
+    <url>https://ge.apache.org</url>
+    <allowUntrusted>false</allowUntrusted>
+    <!--
+    credentials provided by jenkins exposing GRADLE_ENTERPRISE_ACCESS_KEY
+    apache members can create their own on the server
+    -->
+  </server>
+  <!-- options are documented at https://docs.gradle.com/enterprise/maven-extension/#configuring_background_uploading -->
+  <buildScan>
+    <backgroundBuildScanUpload>#{isFalse(env['CI'])}</backgroundBuildScanUpload>
+    <!-- Always publish build scans for CI, only if requested locally -->
+    <publish>#{env['CI'] == null ? 'ON_DEMAND' : 'ALWAYS'}</publish>
+    <capture>
+      <goalInputFiles>false</goalInputFiles> <!-- To be enabled locally when debugging goal caching or if we want to enable predictive test selection -->
+      <buildLogging>false</buildLogging> <!-- disabled by default for privacy and performance, to be discussed -->
+      <testLogging>false</testLogging> <!-- disabled by default for privacy and performance, to be discussed -->
+    </capture>
+    <links>
+      <link>
+        <name>Apache James github</name>
+        <url>https://github.com/apache/james-project/</url>
+      </link>
+      <link>
+        <name>Apache James project website</name>
+        <url>https://github.com/apache/james-project/</url>
+      </link>
+    </links>
+    <obfuscation>
+      <!-- Use Spring Expression Language. -->
+      <!-- <hostname>#{isTrue(env['CI']) ? hostname : 'Local agent'}</hostname> -->
+      <hostname>#{isTrue(env['CI']) ? 'jenkins' : 'Local agent'}</hostname>
+      <ipAddresses>#{sha512(ipAddresses)}</ipAddresses>
```

Comment:
it seems quite simple to reverse a sha512 of an ipAddress by computing the 4 billion possible values.
I even expect this table to be already available somewhere, so it's not much of an obfuscation to me.
For precomputed tables, we could at least add a salt.
WDYT?

Reply:
I didn't think too much about it when copy pasting the sample from the documentation :D 
Also I mostly use ipv6 (vpn) which makes the rainbow table much less obvious ;)

more serioulsy I stumbled on the infra wiki recommandations https://cwiki.apache.org/confluence/display/INFRA/Project+Onboarding+Instructions+for+Develocity and applied that